plugins/gitopen.lua

```diff
@@ -0,0 +1,43 @@
+-- mod-version:3
+local core = require "core"
+local command = require "core.command"
+local common = require "core.common"
+
+
+local function cmd(cmd)
+  local fp = io.popen(cmd, "r")
+  local res = fp:read("*a")
+  fp:close()
+  return res
+end
```

Comment:
I haven't tested the plugin yet, but I'd suggest using our `process` API instead of relying on `io.popen`.
The main reason is that the `process` API allows async operations, so that the UI doesn't block.

You can find some example of `process` usage in a few plugins in this repo (`gitstatus`).
The API documentation is at https://github.com/lite-xl/lite-xl/blob/c29b1c2cb9b50f11cfe3b3bd6a9d59ff0fd8488d/docs/api/process.lua.